src/widgets/health-monitoring/cluster/HealthMonitoringClusterTile.tsx

```diff
@@ -1,24 +1,23 @@
-import { Accordion, Center, Flex, Group, RingProgress, Stack, Text } from '@mantine/core';
+import { Card, Accordion, Center, Flex, Group, RingProgress, Stack, Text } from '@mantine/core';
 import {
   IconBrain,
   IconCpu,
   IconCube,
   IconDatabase,
   IconDeviceLaptop,
   IconServer,
+  IconAlertTriangle,
+  IconCloudDownload,
+  IconHeartRateMonitor,
```

Comment:
Did you forget to remove those (and line 13) after testing?
Don't forget to run "Organize Imports" and "Format Document".
(Formatting should be done using the "prettier" extension)